### PR TITLE
Added test for signing in as an existing user

### PR DESCRIPTION
**Pull request by rbillings**
## Because

\* This adds coverage for testing the ability to log in twice

## This commit

\* adds a test for signing in an existing user

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).

### CODE REVIEW

**Comment by rbillings** [comment]
@vbudhram r?

File: packages/fxa-content-server/tests/functional/sign_in.js [hunk 1]
```diff
@@ -152,6 +152,34 @@ registerSuite('signin', {
...
     },
 
+    'signin as an existing user': function () {
+      return (
+        this.remote
+          .then(createUser(email, PASSWORD, { preVerified: true }))
+          .then(openPage(ENTER_EMAIL_URL, selectors.ENTER_EMAIL.HEADER))
+          .then(fillOutEmailFirstSignIn(email, PASSWORD))
+
+          // success is seeing the error message.
```
Comment (vbudhram) [inline packages/fxa-content-server/tests/functional/sign_in.js:162]: I think this should say seeing the header element is success?

Comment (rbillings) [reply packages/fxa-content-server/tests/functional/sign_in.js]: Agreed!

**Comment by rbillings** [comment]
@vbudhram - if you approve of the change, would you please merge?

**Comment by lmorchard** [comment]
Looks like this needs a GPG signature before we can merge... I can't remember, is there an easy way for one of us core devs to do that or does the original PR branch need a signed commit?

**Comment by meandavejustice** [comment]
Here is the post I always reference when I need to setup a new machine, https://micropipes.com/blog/2016/08/31/signing-your-commits-on-github-with-a-gpg-key/ (it's from wil's blog)

**Comment by lmorchard** [comment]
Yeah, but I was kind of hoping we didn't have to force rbillings to set up GPG if not already setup

**Comment by rbillings** [comment]
My last key expired, so I created a new GPG key and added it.

**Comment by rbillings** [comment]
Since half of this was done unsigned it seems to be stuck in a weird state so I'm closing and re-doing it,

**Pull request closed without merging**